src/utils.ts

```diff
@@ -70,6 +75,17 @@ const converted = (s: string): string => {
     return s;
 };
 
+const isString = (value: any): boolean => typeof value === 'string';
+const isNumber = (value: any): boolean => typeof value === 'number';
+const isFunction = (fn: any): boolean => typeof fn === 'function';
+
+// TODO: Refactor this to a regex
+const isDataPlanSlug = (str: string): boolean =>
+    isString(str) && str === slugify(str, { replacement: '_', lower: true });
```

Comment:
Why are we adding `replacement` here?  If they pass a string to us with spaces, we should be denying it, so I'm unclear how your tests with the spaces are passing. I must be misled?

Reply:
I think I was doing that to better support testing for kabob case, but it doesn't make a difference. I can remove.